ompi/mca/mtl/psm2/mtl_psm2_component.c

```diff
@@ -362,6 +345,12 @@ ompi_mtl_psm2_component_init(bool enable_progress_threads,
     int	verno_major = PSM2_VERNO_MAJOR;
     int verno_minor = PSM2_VERNO_MINOR;
     int local_rank = -1, num_local_procs = 0;
+#if OPAL_CUDA_SUPPORT
+    int ret;
+    char *cuda_env;
+    glob_t globbuf;
```

Comment:
You probably need to `#include <glob.h>` to guarantee that this will work.

Is PSM/PSM2 Linux-only?  I.e., do you need to add a test for `glob()` and/or `<glob.h>` to this component's `configure.ac`?

Reply:
`glob.h` is already included. (There is an existing usage of `glob()` in line 279). PSM2 is Linux only and `glob.h` is included in glibc-headers package. So, don't think we need a check for this in `configure.ac`

Reply:
Oops -- missed that.  You're right.